tmt/utils.py

```diff
@@ -128,6 +128,47 @@ def unrooted(self) -> 'Path':
         return self
 
 
+def configure_optional_constant(default: Optional[int], envvar: str) -> Optional[int]:
+    """
+    Deduce the actual value of a global constant which may be left unset.
+
+    :param default: the default value of the constant.
+    :param envvar: name of the optional environment variable which would
+        override the default value.
+    :returns: value extracted from the environment variable, or the
+        given default value if the variable did not exist.
+    """
+
+    if envvar not in os.environ:
+        return default
+
+    try:
+        return int(os.environ[envvar])
+
+    except ValueError as exc:
+        raise tmt.utils.GeneralError(
+            f"Could not parse '{os.environ[envvar]}' as integer.") from exc
```

Comment:
What about mentioning the variable name in the error message as well?

Reply:
Sure, added in 7a245bef

Reply:
Good, thanks, added to the other function as well in 297b6439.